package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "prop-types": "^15.8.1",
```

Comment:
Why installing prop-types if not using them?
suggestion to delete this dependency.